src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalType.java

```diff
@@ -1,8 +1,8 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
+import java.util.HashMap;
 import java.util.TreeSet;
-import java.util.Hashtable;
 //configuration: default
-public class InputIllegalType {
+public class InputIllegalType implements InputIllegalTypeSuper {
```

Comment:
Please combine 2 inputs into 1. It is better to make inputs completely isolated otherwise it can cause issues when one has to be modified and it affects the other unintentionally.

Reply:
I don't understand. How should I combine two inputs into one?

Reply:
Example:
````
public class InputIllegalType implements IllegalTypeSuper {
}
interface IllegalTypeSuper {
}
````

Unless you are testing something in the import statements that you can't use some existing class, we prefer to have input files isolated to 1 file and not depend on another.

Reply:
So you mean the super class should be within the same file, right?

Reply:
If so, this is changed now

Reply:
Yes.